OneCommunityGlobal/HGNRest
# Leon and Oleksandr add interactive map

# Description
See PR   #[1503](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1503)

## Related PRS :
To test this backend PR you need to checkout the #[1503](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1503) frontend PR.
…

## Main changes explained:
- added map location controller, router and mongoose model
- changed user profile model to handle location object

## How to test:
1.get the API key from https://opencagedata.com/
2. add to env same API key from step 1:
TIMEZONE_PREMIUM_KEY=1234
TIMEZONE_COMMON_KEY=1234
 check into current branch
3. do `npm install` and `npm run build && npm start` to run this PR localy
